### DISCUSSION TITLE
Amplipi Behavior When Changing Away from LMS Stream

### DISCUSSION THREAD

**Discussion by Jizzay1** [micro-nova/AmpliPi#426] [Q&A] [answered]
I'm running into interesting behavior when changing Amplipi source inputs when it comes to the LMS player that is running on the Amplipi.

- With Amplipi LMS stream running on Amplipi (v0.1.9+lms-beta2) my Home Assistant (HA) based Logitech Media Server (LMS) can see the player, connects to it, and plays as expected
- I change the stream for that sournce on the Amplipi from "lms" to anything other stream (such as an RCA input or Airplay ) and can play that stream fine.
- i switch back to Amplipi "lms" stream and now my HA based LMS server sees that that the Amplipi "lms" stream is now part of it's own Amplipi LMS Server called "amplipi".  i can't play to audio to immediately and have to go into my HA LMS server and select that 
"amplipi" server and "move" that player back over to my HA LMS server.
- i have to repeat the "move" back to HA LMS server each time i switch streams on that amplipi source.
Any thoughts?

**Comment by linknum23** [comment, marked as answer]
I think I understand what is happening here. There are 2 LMS servers on your network. One running on AmpliPi and one running elsewhere on your network.


Can you ssh into your amplipi and uninstall the LMS server? Below is the command needed to run.

    sudo apt remove  logitechmediaserver

Good Luck! Let us know if this works.

**Reply by Jizzay1** [reply]
thanks for looking into this so quickly!

my first time SSH'ing into the Amplipi.  username and password?

**Comment by linknum23** [comment]
We have some instructions for that somewhere. Try this:

   1. use an ssh client such as Putty to ssh into ***@***.***
   2. use the username: pi
   3. use the password found on amplipi’s display, you will have to touch
   the display to show it
